code/modules/client/preference/loadout/loadout_suit.dm

```diff
@@ -204,3 +204,15 @@
 	display_name = "mantle, research director"
 	path = /obj/item/clothing/suit/mantle/labcoat
 	allowed_roles = list("Research Director")
+
+//Robes!
+
+/datum/gear/suit/wizard
```

Comment:
Yeahhh I don't think so.

I'd rather not make silly costumes even *more* common.

Reply:
is it just the wizard robes or the witch one too? That one actually looks nice on some people. Also what about the hats?